roles/ruby_s/defaults/main.yml

```
@@ -0,0 +1,6 @@
+---
+# defaults file for roles/ruby
+ruby_version: "{{ ruby_version_override | default(ruby_version_default) }}"
+ruby_version_default: "ruby-2.7.0"
+ubuntu_ruby_version: "{{ ubuntu_ruby_version_override | default(ubuntu_version_default) }}"
```

Typo here in the default setting:
```suggestion
ubuntu_ruby_version: "{{ ubuntu_ruby_version_override | default(ubuntu_ruby_version_default) }}"
```
Though I wonder if we can just set the default to ruby2.7 directly . . .